tsconfig.base.json

```diff
@@ -6,7 +6,6 @@
     "declaration": false,
     "moduleResolution": "node",
     "noUnusedLocals": true,
-    "emitDecoratorMetadata": true,
```

Comment:
Removing this option fixes the coverage bug, see: https://github.com/nrwl/nx/issues/12132